components/runners/ambient-runner/ambient_runner/bridges/claude/memory_tools.py

```
@@ -0,0 +1,319 @@
+
+            file_path = args.get("file_path")
+            category = args.get("category")
+            if file_path or category:
+                findings = api_client.list_findings(
+                    intel["id"], file_path=file_path, category=category
+                )
+                result["findings"] = findings.get("items", [])
+                result["findings_count"] = findings.get("total", len(result["findings"]))
+
+            return _tool_response(result)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)